core/src/main/java/org/apache/iceberg/ManifestGroup.java

```diff
@@ -325,6 +351,9 @@ private static CloseableIterable<FileScanTask> createFileScanTasks(
         entry -> {
           DataFile dataFile = entry.file().copy(ctx.shouldKeepStats());
           DeleteFile[] deleteFiles = ctx.deletes().forEntry(entry);
+          ctx.scanMetrics().totalFileSizeInBytes().increment(dataFile.fileSizeInBytes());
```

Comment:
Should we have a similar metrics for total delete file size?

Reply:
good point, added